src/sentry/utils/silo/decorate_unit_tests.py

```diff
@@ -117,6 +117,7 @@ def __init__(self, cases: Iterable[TestCaseFunction]) -> None:
 
     @cached_property
     def file_map(self) -> Mapping[TestCaseFunction, List[str]]:
+        groups: dict[str | None, dict[str, list[TestCaseFunction]]]
         groups = defaultdict(lambda: defaultdict(list))
```

Comment:
Couldn't these be combined into one line?

Reply:
when combined into one line it becomes 4 lines due to black